ntbs-service/DataMigration/NotificationMapper.cs

```diff
@@ -60,37 +65,77 @@ public NotificationMapper(IConfiguration _configuration)
             connectionString = _configuration.GetConnectionString("migration");
         }
 
-        public async Task<IEnumerable<Notification>> Get(string notificationId)
+        public async Task<List<List<Notification>>> GetByDate(DateTime cutoffDate)
         {
             using (var connection = new SqlConnection(connectionString))
             {
                 connection.Open();
+                var whereClause = string.Format(WhereClauseByDate, $@"'{cutoffDate.ToString("s")}'");
+                var queryWithWhereClause = string.Format(NotificationsQuery, whereClause);

```

Comment:
This is currently slightly open to sql injection - e.g. a row in the import of the format `X')); DROP TABLE Y; etc.` 
It may well be that SqlConnection has safeguards against multiple queries... but not sure.